streampark-console/streampark-console-webapp/src/views/flink/app/components/AppDetail/DetailTab.vue

```diff
@@ -430,6 +465,12 @@
       <TabPane key="6" tab="Option Logs" v-if="tabConf.showOptionLog">
         <BasicTable @register="registerLogsTable">
           <template #bodyCell="{ column, record }">
+            <template v-if="column.dataIndex == 'optionName'">
+              <Tag color="#52c41a" v-if="record.optionName == 0"> Release </Tag>
```

Comment:
We recommend using enumerations to standardize the meaning of data. Here, you can refer to the 220 lines of code in streampark-console-webapp/src/views/flink/app/View. vue. Of course, writing this currently does not affect program operation

Reply:
nice suggestion, done.

Reply:
nice suggestion, done.